src/scripts/main.js

```diff
@@ -1 +1,25 @@
 'use strict';
+
+const section = document.querySelector('.chess');
```

Comment:
Move this line inside `DOMContentLoaded` because `section` inside it could be undefined.